FRAMEWORKS.yml

```diff
@@ -23,6 +23,10 @@ ruby:
     website: hanamirb.org
     version: "1.3"
     language: "2.5"
+  agoo::
+    website: github.com/ohler55/agoo
+    version: "2.5.1"
```

Comment:
could you specify X.Y (I mean using `2.5` instead of `2.5.1`)
using `2.5` is clearer for everyone :heart: